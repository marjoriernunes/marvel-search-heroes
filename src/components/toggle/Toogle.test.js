import {render} from '@testing-library/react';
import ToggleButton from './Toggle';

test('renders correctly', () =>{
    const utils = render(<ToggleButton/>)
    expect(utils.container).toMatchSnapshot()
});
