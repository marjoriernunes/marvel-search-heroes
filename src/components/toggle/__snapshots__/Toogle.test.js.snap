// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    class="wrg-toggle"
    data-testid="toggle-div"
  >
    <div
      class="wrg-toggle-container"
    />
    <div
      class="wrg-toggle-circle"
    />
    <input
      aria-label="Toggle Button"
      class="wrg-toggle-input"
      type="checkbox"
    />
  </div>
</div>
`;
